<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoWindow</class>
 <widget class="C8CommonWindow" name="UserInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserInfoWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>362</width>
       <height>232</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="topMargin">
             <number>25</number>
            </property>
            <property name="bottomMargin">
             <number>25</number>
            </property>
            <item>
             <widget class="QLabel" name="label_useInfoHeadIcon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>122</width>
                <height>122</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>122</width>
                <height>122</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_useInfoTea">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>25</number>
          </property>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_close">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>17</number>
            </property>
            <property name="leftMargin">
             <number>30</number>
            </property>
            <property name="topMargin">
             <number>7</number>
            </property>
            <property name="rightMargin">
             <number>44</number>
            </property>
            <property name="bottomMargin">
             <number>25</number>
            </property>
            <item>
             <widget class="QLabel" name="label_teacher_id">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>12</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string>userinfo_ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_teacher_nickname">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>12</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string>userinfo_NickName</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_teacher_name">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>12</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string>userinfo_Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_teacher_sex">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>12</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string>userinfo_Sex</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_teacher_flag">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>12</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string>userinfo_Flag</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>17</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_homepage_pic">
                  <property name="minimumSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_home_page">
                  <property name="text">
                   <string>homePage</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_swicher_pic">
                  <property name="minimumSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16</width>
                    <height>16</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_whisper">
                  <property name="text">
                   <string>privateChat</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>C8CommonWindow</class>
   <extends>QWidget</extends>
   <header>c8commonwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
